### PR TITLE
Adapted LMCache Test to new version

### PR DESCRIPTION
**Pull request by qyy2003**
Update installation script in the current lmcache-tests repo to new LMCache (install from source)
Update the command line for Local bootstraper to use the latest lmcache_vllm

Python 3.10/3.11/3.12 are all verified。
Tested on `test_lmcache_local_cpu` and `test_lmcache_remote_disk`

### CODE REVIEW

**Comment by ApostaC** [comment]
Thanks! Will start reviewing soon!

**Review by ApostaC — COMMENTED** [review]
Looks good to me! Thanks!

Just one minor comment

File: configs/lmcache_local_cpu old.yaml
[the remark this answers is not in the document]
Comment (qyy2003) [reply configs/lmcache_local_cpu old.yaml:1]: this is the original version. I copied a new version from LMCache repository and kept the old one. I guess we should remove this when we update all the config files. Maybe it's not bad to keep it there.

[the remark this answers is not in the document]
Comment (ApostaC) [reply configs/lmcache_local_cpu old.yaml:1]: I see your point. The goal here is to make the filenames clean and informative. The `lmcache_local_cpu old` could be a bit confusing.
If your change fixes the errors in the system, feel free to drop the old version. 
If your change is just for some local test, please exclude those from the commit.

Let's know if you have any other concerns, thanks!

**Comment by qyy2003** [comment]
@ApostaC I've deleted it.~

**Review by ApostaC — APPROVED** [review]
cool, LGTM!

**Pull request merged**